client/client_test.go

```diff
@@ -352,7 +353,7 @@ func (s *ClientSuite) TestNewRoot(c *C) {
 		for _, id := range ids {
 			key := client.db.GetKey(id)
 			c.Assert(key, NotNil)
-			c.Assert(key.IDs(), DeepEquals, ids)
+			c.Assert((*key).IDs(), DeepEquals, ids)
```

Comment:
go help: key is an implementation of an interface so i think i had to dereference this
maybe instead i should return the object from GetKey along with an error rather than checking IsNil?

Reply:
changed to key, err